coffeehouse/lydia/session.py

```diff
@@ -8,10 +8,10 @@ def __init__(self, data, client):
         """
 
         self._client = client
-        self.id = data["session_id"]
-        self.language = data["language"]
-        self.available = data["available"]
-        self.expires = data["expires"]
+        self.id = data['session_id']
+        self.language = data['language']
+        self.available = data['available']
+        self.expires = data['expires']
```

Comment:
Note that a session now contains more information about the session itself.

https://docs.intellivoid.net/coffeehouse/v1/lydia/get_session
https://docs.intellivoid.net/coffeehouse/v1/lydia/get_session_attributes

Reply:
OMG>>>>>>

Reply:
yeah let me add more commits to lydia. i didnt play around it (just my ide auto refractoring this)